### PR TITLE
 66430 (11.2-U3): Clarify replication process and interruptions.

### PR DESCRIPTION
**Pull request by Mrt134**
Clean up a couple extra commits that crept into my branch.
HTML build test: no issues.

### CODE REVIEW

File: userguide/tasks.rst [hunk 1]
```diff
@@ -809,23 +809,25 @@ system, with that system sometimes being at a different physical
...
-.. note:: The target dataset on the receiving system is automatically
-   created in read-only mode to protect the data. To mount or browse
-   the data on the receiving system, create a clone of the snapshot
-   and use the clone. Clones are created in read/write mode, making it
-   possible to browse or mount them. See :ref:`Snapshots` for more
-   information on creating clones.
+data and a destination system where the data will be replicated. The
+destination system is prepared to receive replicated data when a
+:ref:`periodic snapshot <Periodic Snapshot Tasks>` of the source system
+data is created and a replication task is created.
```
Comment (a-stjohn) [inline userguide/tasks.rst]:
```suggestion
data and a replication task are created.
```

File: userguide/tasks.rst [hunk 2]
```diff
@@ -809,23 +809,25 @@ system, with that system sometimes being at a different physical
...
+
+When snapshots are automatically created on the source computer, they
+are automatically replicated to the destination computer. First-time
+replication tasks can take a long time to complete as the entire
+snapshot must be copied to the destination system. Replicated data is
+not visible on the receiving system until the replication task
+completes. Later replications only send the snapshot changes to the
+destination system. Interrupting a running replication will require
```
Comment (a-stjohn) [inline userguide/tasks.rst]:
```suggestion
destination system. Interrupting a running replication requires
```

File: userguide/tasks.rst [hunk 3]
```diff
@@ -809,23 +809,25 @@ system, with that system sometimes being at a different physical
...
-   and use the clone. Clones are created in read/write mode, making it
-   possible to browse or mount them. See :ref:`Snapshots` for more
-   information on creating clones.
+data and a destination system where the data will be replicated. The
+destination system is prepared to receive replicated data when a
+:ref:`periodic snapshot <Periodic Snapshot Tasks>` of the source system
+data and a replication task are created.
+
+When snapshots are automatically created on the source computer, they
+are automatically replicated to the destination computer. First-time
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
are replicated to the destination computer. First-time
```

File: userguide/tasks.rst [hunk 4]
```diff
@@ -809,23 +809,25 @@ system, with that system sometimes being at a different physical
...
+replication tasks can take a long time to complete as the entire
+snapshot must be copied to the destination system. Replicated data is
+not visible on the receiving system until the replication task
+completes. Later replications only send the snapshot changes to the
```
Comment (wblock) [inline userguide/tasks.rst]:
```suggestion
completes. Later replications only send the changes to the
```

File: userguide/tasks.rst [hunk 5]
```diff
@@ -809,23 +809,25 @@ system, with that system sometimes being at a different physical
...
-
-
-.. note:: The target dataset on the receiving system is automatically
-   created in read-only mode to protect the data. To mount or browse
-   the data on the receiving system, create a clone of the snapshot
-   and use the clone. Clones are created in read/write mode, making it
-   possible to browse or mount them. See :ref:`Snapshots` for more
-   information on creating clones.
+data and a destination system where the data will be replicated. The
+destination system is prepared to receive replicated data when a
```
Comment (wblock) [inline userguide/tasks.rst]: When a :ref:`periodic snapshot <Periodic Snapshot Tasks>` of the selected dataset occurs, the replication task copies the data to the destination system.

**Pull request merged**